design/one-pager-package-format-v2.md

```diff
@@ -0,0 +1,248 @@
+will now only install a specific kind of controller - a Crossplane provider. It
+is possible that both the maintainer of the provider and the platform operator
+will need to influence how the provider is deployed; e.g. by specifying env
+vars, health checks, replica counts, or node selectors. Currently all providers
+use a minimal `install.yaml`; they do not make use of their ability to influence
+how the provider is run. This document recommends that the above `Provider`
+types be extended when and as appropriate to configure these concerns.
+
+### Packaging a Configuration
+
+A packaged Crossplane configuration must be an OCI image consisting of:
+
+1. A `crossplane.yaml` file containing a `kind: Configuration` at its root.
+1. One or more `CompositeResourceDefinition` and/or `Composition` resources.
+1. Optional additional metadata that is opaque to Crossplane in a `.crossplane`
+   directory at the root of the package.
+
+Below is an example `Configuration`, at `v1alpha`:
+
+```yaml
+# Required. Must be as below.
+apiVersion: meta.packages.crossplane.io/v1alpha1
+# Required. Must be as below.
+kind: Configuration
+# Required. Note that Crossplane is aware only of the name and annotations
+# metadata fields. Other fields (e.g. labels) will be preserved but opaque.
+metadata:
+  # Required. Must comply with Kubernetes API conventions.
+  name: configuration-example
+  # Optional. Must comply with Kubernetes API conventions. Annotations are
+  # opaque to Crossplane, which will replicate them to the annotations of a
+  # PackageRevision when this package is unpacked, but otherwise ignore them.
+  # Systems such as package registries may extend this specification to require
+  # or encourage specific annotations.
+  annotations:
+    company: Upbound
+    maintainer: Nic Cope <negz@upbound.io>
+    keywords: cloud-native, kubernetes, example
+    source: github.com/crossplane-contrib/config-example
+    license: Apache-2.0
+    description: |
+      The Example configuration adds example resources to Crossplane.
+    provider: example
+# Required.
+spec:
+  dependsOn:
+    # Required. Specifies an OCI image containing a package dependency. This key
+    # may be either 'provider' or 'configuration'. This is sugar; in either case
+    # the package manager determines whether the depencency is really a Provider
+    # or a Configuration by unpacking it and inspecting its kind.
+  - provider: example/provider-example
+    # Required. Will be extended to support version ranges in future, but
+    # currently treated as a specific version tag.
+    version: v0.1.0
+    # Required. Specifies an OCI image containing a package dependency. This key
+    # may be either 'provider' or 'configuration'. This is sugar; in either case
+    # the package manager determines whether the depencency is really a Provider
+    # or a Configuration by unpacking it and inspecting its kind.
+  - configuration: example/some-dependency
```

Comment:
> not sure what your current thinking is on the future of such a feature.

The idea is still in the wings, but it seems far enough off that I'd prefer not to try sketch out the UX until we have more time to design it. So I think as it relates to this one pager the only important thing is that we feel like we could extend this API type to support declaring a dependency on a GVK rather than a provider or configuration.